DesktopModules/Vanjaro/Core/Library/Managers/ThemeManager.cs

```diff
@@ -21,21 +21,21 @@ public static partial class Managers
     {
         public class ThemeManager
         {
-            public static string GetCurrentThemeName(int? PortalID = null)
+            public static Theme GetCurrent(int? PortalID = null)
```

Comment:
This should not be a method but a singleton. For example, ThemeManager.CurrentTheme.Name

Reply:
We require a portalid when the calls from IUpgradeable.

Reply:
@narensingh008 @mksmanish79 See mee to discuss these changes.